tools/doc/README.md

```diff
@@ -35,7 +35,7 @@ A description of the function.
 
 ## module.someNewFunction(x)
 <!-- YAML
-added: REPLACEME
+added: v0.10.0
```

Comment:
This change should not land. Going to leave a red x here, but dismiss it if this commit/change is removed.